frontend/src/Components/Testimonial/Cards.js

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import Cardtemplate from "./Cardtemplate";
+import ocomma from "./../../images/ocomma.png";
+import scomma from "./../../images/scomma.png";
+import andrea from "./../../images/andrea.png";
+
+const details = [{
```

Comment:
Make details object as a separate component in `Test_data` folder and fetch from there.